comp/otelcol/otlp/components/processor/infraattributesprocessor/common.go

```diff
@@ -0,0 +1,74 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+package infraattributesprocessor
+
+import (
+	"fmt"
+	"strings"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	conventions "go.opentelemetry.io/collector/semconv/v1.21.0"
+
+	"github.com/DataDog/datadog-agent/comp/core/tagger/tags"
+	"github.com/DataDog/datadog-agent/comp/core/tagger/types"
+)
+
+var unifiedServiceTagMap = map[string]string{
+	tags.Service: conventions.AttributeServiceName,
+	tags.Env:     conventions.AttributeDeploymentEnvironment,
```

Comment:
@dineshg13 should we be preventing the override for `deployment.environment` or `deployment.environment.name` here? Or should we be doing both

Reply:
We should do mapping for both.

Reply:
I would create OTel -> DD semantics map as multiple Otel semantics can map to single DD semantic